### PR TITLE
Implement more of GlowContainer

### PR DESCRIPTION
**Pull request by Pr0methean**

### CODE REVIEW

File: src/main/java/net/glowstone/block/entity/state/GlowContainer.java [hunk 1]
```diff
@@ -1,24 +1,36 @@
 package net.glowstone.block.entity.state;
 
 import com.destroystokyo.paper.loottable.LootableBlockInventory;
+import java.util.Collections;
```
Comment (aramperes) [inline src/main/java/net/glowstone/block/entity/state/GlowContainer.java]: Unused imports: `java.util.Collections`, `java.util.Set`, `lombok.Data`, `lombok.EqualsAndHashCode`.

File: src/main/java/net/glowstone/block/entity/state/GlowContainer.java [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
     private final AtomicLong lootTableSeed = new AtomicLong(0);
+    private final AtomicReference<String> lootTable = new AtomicReference<>(null);
+    private final Map<UUID, Long> playersWhoHaveLooted = new ConcurrentHashMap<>();
+    @Getter
+    @Setter
     private String lock;
+    @Getter
+    @Setter
     private String customName;
     public GlowContainer(GlowBlock block) {
```
Comment (aramperes) [inline src/main/java/net/glowstone/block/entity/state/GlowContainer.java]: Add a blank line before the constructor.

**Review by aramperes — DISMISSED** [review]

File: src/main/java/net/glowstone/block/entity/state/GlowContainer.java [hunk 3]
```diff
@@ -29,22 +37,23 @@ public GlowContainer(GlowBlock block) {
...
     }
 
     @Override
     public String setLootTable(String name) {
-        return null;
+        return setLootTable(name, 0);
     }
 
     @Override
     public String setLootTable(String s, long l) {
```
Comment (aramperes) [inline src/main/java/net/glowstone/block/entity/state/GlowContainer.java]: Can you give the variables proper names here?

Comment (Pr0methean) [reply src/main/java/net/glowstone/block/entity/state/GlowContainer.java]: Done in f2d3ca7

**Review by aramperes — APPROVED** [review]

**Review by mastercoms — APPROVED** [review]

**Pull request merged**